pallets/compliance-manager/src/lib.rs

```diff
@@ -892,8 +899,8 @@ impl<T: Config> Pallet<T> {
 impl<T: Config> ComplianceFnConfig for Pallet<T> {
     fn is_compliant(
```

Comment:
Changing this to an `Option` allows for checking senders and receivers independently